src/h_vialib/secure/nonce.py

```diff
@@ -0,0 +1,29 @@
+"""Create values we can check we created."""
+
+from uuid import uuid4
+
+from h_vialib.secure.token import SecureToken
+
+
+class RandomSecureNonce(SecureToken):
+    """A random token we can verify we created with an expiry time.
+
+    This token should be different every time you create it.
+    """
```

Comment:
```suggestion
class RandomSecureNonce(SecureToken):
    """A source of random tokens we can verify we created with an expiry time."""
```

I think the comment about the token being different every time belongs on the `create()` method, as I think that's the method that returns a different token each time called.

And I think a `RandomSecureNonce` instance if not actually "a random token", it's an object that can generate random tokens for you.

Reply:
Yeah as you say, they are all factories